i18n/bn.toml

```diff
@@ -0,0 +1,58 @@
+[home]
```

Comment:
This is an additional modification not related to this PR. Would you mind opening a new PR to add Bengali translations? 😃

Reply:
Ah my bad, I was pushing everything to master at 2 am...

Reply:
PR: https://github.com/g1eny0ung/hugo-theme-dream/pull/267